LutronLuxmeterIntegration/src/widget.ts

```diff
@@ -0,0 +1,101 @@
+import { CamOverlayDrawingOptions } from 'camstreamerlib/CamOverlayDrawingAPI';
+import { Painter, PainterOptions, Frame } from 'camstreamerlib/CamOverlayPainter/Painter';
+import type { TResult } from './reader';
+
+const imageWidth = 325;
+const luxmeterHeight = 254;
+const camstreamerHeight = 92;
+
+function toString(value: number): string {
+    if (value === 0 || isNaN(value)) {
+        return 'ERROR';
+    } else {
+        const text = value.toPrecision(4);
+        return text.substring(0, 6);
+    }
+}
+function initialisePainter(opt: PainterOptions, coOpt: CamOverlayDrawingOptions, ...frames: Frame[]) {
```

Comment:
proc to nejsou funkce primo tridy widget?

Reply:
Jsem líný psát this, když to není potřeba.

Reply:
mi uplne neprijde jako dostatecny duvod.. ten kod se hur, alespon podle me

Reply:
Nebudu se hádat, ale myslím si o tom svoje.